lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -343,31 +343,36 @@
       </div>
 
       <div class="container__row">
-          <div class="container__block">
-            <h3 class="inline-alt-text-color"><?php print t('Pics or It Didn&rsquo;t Happen'); ?></h3>
-            <?php if (isset($reportback_copy)): ?>
-              <p class="copy inline-alt-text-color"><?php print $reportback_copy; ?></p>
-            <?php endif; ?>
-
-            <?php if (isset($helpful_tips)): ?>
-              <h3 class="inline-alt-text-color"><?php print t('Helpful Tips'); ?></h3>
-              <div class="with-lists copy inline-alt-text-color">
-                <?php print $helpful_tips; ?>
-              </div>
-            <?php endif; ?>
+        <div class="container__block">
+          <h3 class="inline-alt-text-color"><?php print t('Pics or It Didn&rsquo;t Happen'); ?></h3>
+          <?php if (isset($reportback_copy)): ?>
+            <p class="copy inline-alt-text-color"><?php print $reportback_copy; ?></p>
+          <?php endif; ?>
 
-            <div class="container__row">
-              <div class="container__block">
-                <h2 class="heading -emphasized"><?php print t('Join others making a difference'); ?></h2>
-              </div>
+          <?php if (isset($helpful_tips)): ?>
+            <h3 class="inline-alt-text-color"><?php print t('Helpful Tips'); ?></h3>
+            <div class="with-lists copy inline-alt-text-color">
+              <?php print $helpful_tips; ?>
             </div>
+          <?php endif; ?>
+        </div>
+      </div>
 
-            <?php if(dosomething_reportback_exists($campaign->nid)): ?>
-              <a href="#" data-modal-href="#modal-missing-photos">Is your photo not showing up?</a>
-            <?php endif; ?>
-          </div>
+      <div class="container__row">
+        <div class="container__block">
+          <h2 class="heading -emphasized"><?php print t('Join others making a difference'); ?></h2>
+        </div>
       </div>
 
+      <?php if(dosomething_reportback_exists($campaign->nid)): ?>
+        <div class="container__row">
+          <div class="container__block">
+            <a href="#" data-modal-href="#modal-missing-photos">Is your photo not showing up?</a>
+          </div>
+        </div>
+      <?php endif; ?>
```

Comment:
I don't think you need a separate `container__row` for the "Is your photo not showing up?" content. It relates to the reportback gallery below which the "Join others making a difference" is kind of a heading for. You should be able to put this inside the `container__row` above.